youcan-shop/youcan-themes
# TH-230: Add Products-by-Collection section

## JIRA Ticket

Ticket: [TH-230](https://youcanshop.atlassian.net/browse/TH-230).

## Prerequisites


## QA Steps

* [ ] Go to seller-area
* [ ] make sure you have a collection with products in it
* [ ] Once that's done go to theme editor (Edit Origins)
* [ ] You should see a "products by collection" section
* [ ] Click on it and start customizing it.
* [ ] The customization options should be the following:
	- Collection
	- Title (Section)
	- Description (Section)
	- Header position: Left, Right, Center
	- View Style: Grid, or Slides
	- Text Color
	- Background color
	- Maximum products to show
	- Number of columns on desktop
	- Show link to collection
	- Enable quick add (This setting won’t work since we don’t have neither the cart or the modal logic yet)
	- Section Spacing (Top, bottom)
* [ ] Play with those customization settings, try to break it.

> [!Note]
>
> You might find some responsitivity issues when the section is in "Slides" mode. That's an issue with our core slider and will be fixed in the PR of this [ticket](https://youcanshop.atlassian.net/browse/TH-285). 
> Another issue arose where the contents of the slides are not responsive to user clicks ([Ticket](https://youcanshop.atlassian.net/browse/TH-286)).


[TH-230]: https://youcanshop.atlassian.net/browse/TH-230?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
